cli/tests/integration/common.py

```diff
@@ -0,0 +1,96 @@
+import contextlib
```

Comment:
This is all copy-pasted from `test_routes.py` so that it can be shared across integration tests